src/app.rs

```diff
@@ -873,8 +921,18 @@ impl App {
                 }
                 _ => {}
             }
-        } else if self.delete_dialog_state.is_showing_dd && !self.delete_dialog_state.is_on_yes {
-            self.delete_dialog_state.is_on_yes = true;
+        } else if self.delete_dialog_state.is_showing_dd {
+            match self.delete_dialog_state.selected_signal {
+                KillSignal::KILL(prev_signal) => {
+                    self.delete_dialog_state.selected_signal = match prev_signal - 1 {
+                        0 => KillSignal::CANCEL,
+                        // 32+33 are skipped
+                        33 => KillSignal::KILL(31),
+                        signal => KillSignal::KILL(signal),
+                    };
+                }
+                KillSignal::CANCEL => (),
+            };
```

Comment:
On Windows this is broken, the movement is reversed.  Moving left moves right instead.